build.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs-extra");
+const generateHTMLs = require("./utils/generateHTMLs.js");
+const config = require("./config.js");
+
+function clearBuildPath(buildPath){
+  try {
+    fs.rmSync(buildPath, { recursive: true });
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+function copyStaticFiles(buildPath){
+  try {
+    fs.copySync("./assets", buildPath);
+  } catch (err) {
+    console.log(err);
+  }
+}
+
+(async function() {
```

Comment:
why is that async when all your code is synchronous.

Reply:
It is because I truncate this code from a larger piece, and it contains asynchronous methods. I've removed it now, but it might come back in the future.